### ISSUE TITLE
Cannot build master - missing ohdr.version

### ISSUE THREAD

**Issue by magx2** [openhab/openhab2-addons#4045]
I'm trying to run ```mvn clean install``` on updated master version of OpenHAB addons, but I get this error:

```
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:1.2.0:compile (default-compile) on project org.openhab.binding.airquality: Execution default-compile of goal org.eclipse.tycho:tycho-compiler-plugin:1.2.0:compile failed: Plugin org.eclipse.tycho:tycho-compiler-plugin:1.2.0 or one of its dependencies could not be resolved: Failed to collect dependencies at org.eclipse.tycho:tycho-compiler-plugin:jar:1.2.0 -> org.eclipse.tycho:tycho-compiler-jdt:jar:1.2.0 -> org.eclipse.jdt:org.eclipse.jdt.core:jar:3.13.102 -> org.eclipse.platform:org.eclipse.core.resources:jar:[3.12.0,4.0.0): No versions available for org.eclipse.platform:org.eclipse.core.resources:jar:[3.12.0,4.0.0) within specified range -> [Help 1]
```

I think it might be connected with logs that are shown at the begging of the build:

```
[WARNING] Could not transfer metadata org.openhab:pom-tycho:2.4.0-SNAPSHOT/maven-metadata.xml from/to p2-openhab-deps-repo (https://dl.bintray.com/openhab/p2/openhab-deps-repo/${ohdr.version}): No connector available to access repository p2-openhab-deps-repo (https://dl.bintray.com/openhab/p2/openhab-deps-repo/${ohdr.version}) of type p2 using the available factories WagonRepositoryConnectorFactory
Downloading: https://openhab.jfrog.io/openhab/libs-snapshot/org/openhab/pom/2.4.0-SNAPSHOT/maven-metadata.xml
Downloaded: https://openhab.jfrog.io/openhab/libs-snapshot/org/openhab/pom/2.4.0-SNAPSHOT/maven-metadata.xml (593 B at 1.8 KB/sec)
[WARNING] Could not transfer metadata org.openhab:pom:2.4.0-SNAPSHOT/maven-metadata.xml from/to p2-openhab-deps-repo (https://dl.bintray.com/openhab/p2/openhab-deps-repo/${ohdr.version}): No connector available to access repository p2-openhab-deps-repo (https://dl.bintray.com/openhab/p2/openhab-deps-repo/${ohdr.version}) of type p2 using the available factories WagonRepositoryConnectorFactory
[WARNING] The POM for com.sun.xml.bind:jaxb-core:jar:2.2.11 is invalid, transitive dependencies (if any) will not be available, enable debug logging for more details
[WARNING] The POM for com.sun.xml.bind:jaxb-impl:jar:2.2.11 is invalid, transitive dependencies (if any) will not be available, enable debug logging for more details
[WARNING] No explicit target runtime environment configuration. Build is platform dependent.
```

Maven cannot find this repository ```https://dl.bintray.com/openhab/p2/openhab-deps-repo/${ohdr.version}```. Apparently it didn't change ```${ohdr.version}``` to correct version. 

Any ideas how to fix it?

**Comment by kaikreuzer** [comment]
The `${ohdr.version}`warning is unrelated (It's main message is "No connector available to access repository p2-openhab-deps-repo", so it doesn't matter at all whether this variable is set or not).

The build works smoothly on our Jenkins, so I cannot tell you what's wrong. Did you try to clean your local Maven repo to make sure that no old local artifacts are used?

**Comment by magx2** [comment]
You were right. removing ```~/.m2``` worked. Thanks!

**Comment by BobRak** [comment]
I have had the exact same problem building my smartthings binding in OH2. 
My environment is:
Windows 10, Build 17134

java version "1.8.0_40"
Java(TM) SE Runtime Environment (build 1.8.0_40-b25)
Java HotSpot(TM) 64-Bit Server VM (build 25.40-b25, mixed mode)

Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T11:33:14-07:00)

I was also successful after deleting my .m2 directory. I've been working on OH bindings for 2-1/2 years and this is the first time I've had to delete this directory to be able to compile. There must be something wrong with the maven build scripts. I'm not a Maven expert and it is beyond my ability to diagnose and fix this.  As a binding developer I think it is reasonable to expect the tooling provided by the project to work correctly. I've spent several hours over the past 3 days trying to figure out why I can't do a Maven build. My time would have been better spent working on the binding so I can finish it and deploy it. Thank you @magx2 to logging this issue, I was about to throw the towel in on this effort.
